addon/components/flexberry-file.js

```diff
@@ -1,334 +1,523 @@
 /**
- * @module ember-flexberry
+  @module ember-flexberry
  */
 
 import Ember from 'ember';
 import FlexberryBaseComponent from './flexberry-base-component';
 import { translationMacro as t } from 'ember-i18n';
 
 /**
- * Flexberry file component.
- *
- * @class FlexberryFile
- * @extends FlexberryBaseComponent
+  Flexberry file component.
+
+  @class FlexberryFile
+  @extends FlexberryBaseComponent
  */
 export default FlexberryBaseComponent.extend({
   /**
-   * Class names for component wrapping <div>.
+    Selected file content. It can be used as source for image tag in order to view preview.
+
+    @property _previewImageAsBase64String
+    @private
+    @type String
+    @default null
```

Comment:
Because some settings are still initializing through flexberry-base-component mechanism (from configuration/environment.js) and such setings need to be undefined in component's class.